.github/workflows/build-wheels-pypi.yml

```diff
@@ -0,0 +1,120 @@
+name: Build Cross platform / architecture wheels and upload to PyPi
+# See https://github.com/pypa/cibuildwheel for details
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "test[0-9]+.[0-9]+.[0-9]+"
+
+  # Trigger on request.
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: Build wheel for ${{ matrix.os }}, Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, macos-latest, windows-2022]
+        python-version: ["3.9", 3.10",  "3.11"]
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3.3.0
+
+      # NOTE: Use QEMU block to build Linux aarch64 builds.  Its SUPER SLOW
+      # so it is commented out and left as a reference
+      # see https://cibuildwheel.readthedocs.io/en/stable/faq/#emulation
+      # - name: Set up QEMU
+      #   if: runner.os == 'Linux'
+      #   uses: docker/setup-qemu-action@v2
+      #   with:
+      #     platforms: 'arm64'
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.12.0
+        env:
+          CIBW_PROJECT_REQUIRES_PYTHON: "==${{ matrix.python-version }}.*"
+
+          # Skip musl linux and PyPy versions
+          CIBW_SKIP: "{*-musllinux_*,pp*}"
+
+          # Build only on 64-bit architectures.
+          CIBW_ARCHS_MACOS: "x86_64 arm64"
+
+          # NOTE: Commented out to be used with QEMU block above
+          # CIBW_ARCHS_LINUX: "x86_64 aarch64"
+
+          CIBW_ARCHS_LINUX: "auto64"
+          CIBW_ARCHS_WINDOWS: "auto64"
+
+          # Configure environment variables.
+          MACOSX_DEPLOYMENT_TARGET: "10.14"
+
+          CIBW_BEFORE_BUILD: 'python3 -m pip install cython pytest "setuptools>=67.1.0" && python3 setup.py build_ext --inplace'
+
+          # Test the wheels.
+          CIBW_TEST_COMMAND_MACOS: "pytest {project}/tests/test_thermoanalysis.py -v"
+          CIBW_TEST_COMMAND_LINUX: "pytest {project}/tests/test_thermoanalysis.py -v"
+          CIBW_TEST_COMMAND_WINDOWS: "pytest {project}\\tests\\test_thermoanalysis.py"
```

Comment:
Thoughts on expanding testing back to all tests now that we aren't building on `aarch64`?

Reply:
I don't think it's super important since almost everything runs through the `thermoanalysis.pyx` now.  As long as the CI tests pass, it's all fine to not add back the additional test.

Reply:
Makes sense to me